pom.xml

```diff
@@ -333,35 +361,37 @@ limitations under the License.
     <rudder-version>4.3.1-SNAPSHOT</rudder-version>
     <spring-run-dep-version>4.3.1-SNAPSHOT</spring-run-dep-version>
 
-    <scala-version>2.12.4</scala-version>
+    <scala-version>2.12.6</scala-version>
     <scala-binary-version>2.12</scala-binary-version>
-    <lift-version>3.0.2</lift-version>
+    <lift-version>3.2.0</lift-version>
```

Comment:
changing the major lift version in a minor release doesn't sound like a great idea

Reply:
Since 3.0, lift is using semantic versionning. It's only bug correction and feature addition. Nothing breaking for use, AFAIK. And it's really early in 4.3 cycle, wainting N month to do the change inquires a cost for java 9/10 compat.